integration-tests/suites/base.go

```diff
@@ -183,32 +173,11 @@ func (s *IntegrationTestSuiteBase) RegisterCleanup(containers ...string) {
 	})
 }
 
-func (s *IntegrationTestSuiteBase) GetContainerStats() []ContainerStat {
+func (s *IntegrationTestSuiteBase) GetContainerStats() []executor.ContainerStat {
 	if s.stats == nil {
-		s.stats = make([]ContainerStat, 0)
-
-		logs, err := s.containerLogs(containerStatsName)
-		if err != nil {
-			assert.FailNow(s.T(), "container-stats failure")
-			return nil
-		}
-
-		logLines := strings.Split(logs, "\n")
-		for i, line := range logLines {
-			var stat ContainerStat
-
-			_ = json.Unmarshal([]byte(line), &stat)
-
-			if stat.Name != "" {
-				s.stats = append(s.stats, stat)
-			} else {
-				log.Warn("missing name for stat line %d of %d", i, len(logLines))
-			}
-		}
-
-		s.cleanupContainers(containerStatsName)
+		s.statsManager.Stop()
+		s.stats = append(s.stats, s.statsManager.GetStats()...)
```

Comment:
Wait, why do we need to copy the stats? Couldn't this entire method just be...?
```go
s.statsManager.Stop()  // Only really stops the manager once.
return s.statsManager.GetStats()
```

Reply:
Good point, done